packages/simplexmq_io/test/transport_test.dart

```
@@ -0,0 +1,38 @@
+import 'package:simplexmq/simplexmq.dart';
+import 'package:simplexmq/src/buffer.dart';
+import 'package:simplexmq/src/crypto.dart';
+import 'package:simplexmq/src/rsa_keys.dart';
+import 'package:simplexmq_io/simplexmq_io.dart';
+import 'package:test/test.dart';
+
+final keyHash =
+    decode64(encodeAscii('pH7bg7B6vB3uJ1poKmClTAqr7yYWnAtapnIDN7ypKxU='));
+void main() {
+  group('SMP transport (expects SMP server on localhost:5223)', () {
+    test('establish connection', () async {
+      final conn = await SocketTransport.connect('localhost', 5223);
```

The test should fail gracefully if the server is not available